package.json

```diff
@@ -33,9 +33,11 @@
   },
   "scripts": {
     "gulp": "gulp",
+    "prepare": "gulp",
```

Comment:
What's the difference between prepare and prestart? Do we need both? And can you put them in alphabetical order?

Reply:
https://docs.npmjs.com/cli/v8/using-npm/scripts documents the built-in npm scripts. `npm install` will run `prepare` but `npm start` won't, so adding as `prestart` ensures that we run the `gulp` build before launching the web server and the build watcher.

I'll alphabetize.